src/com/company/Main.java

```diff
@@ -42,9 +41,12 @@ public static void main(String[] args) {
             case 17:
                 zeroIfNegative();
                 break;
+            case 5:
+            	find_smallest_prime();
+            	break;
             case 18: // Execute All
                 // Add Your Function Here
-                zeroIfNegative();
```

Comment:
this is an execute all function,
so literally it will execute all functions, why would u remove others function ?